spec/models/category_spec.rb

```diff
@@ -17,18 +19,17 @@
     end
 
     it "must return 6 videos at most" do
-      stargate = Fabricate.times(7,:video, category: category)
+      Fabricate.times(7,:video, category: category)
       expect(category.recent_videos.count).to eq(6)
     end
 
     it "must be ordered by created_at ascending(last video first)" do
-      stargate_seasons = 7
-      seasons_array = []
-      stargate_seasons.times do |season|
-        stargate = Fabricate(:video, category: category)
-        seasons_array << stargate
+      videos_array = []
+      7.times do |season|
+        video = Fabricate(:video, category: category)
+        videos_array << video

```

Comment:
Thanks. It´s true, in fact i changed and you write less code this way. Apart from been more accurate as your say.